### PR TITLE
[ENHC0010020] Second part of Delete multiple samples to update import of utilities/form file

### PR DESCRIPTION
**Pull request by deepsidhu85**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR removes relative import of the app/javascript/utlilities/form directory and instead adds it to the importmap and is imported in js controllers with `utlilities/form`

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

1. Create a project, add 10 samples
2. Try out the clone/export/tranfer/delete files 
3. Verify that the dialog loads correctly and lists the files that were selected
4. Verify with the sample -> attachments as well
5. Verify there are no console errors in the browser

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/javascript/application.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails";
 import "controllers";
+import "utilities/form";
```
Comment (joshsadam) [inline app/javascript/application.js:4]: **question:** Do we need to add this here, by doing so it gets loaded onto every page

**Review by joshsadam — COMMENTED** [review]
Just a question

File: app/javascript/application.js (hunk 1 above)
Comment (deepsidhu85) [reply app/javascript/application.js]: Looks like it does not need to be here. Thanks for catching this. Updated in 8ee769258c40d65ed8637d4ffad500a93608af7f

**Review by JeffreyThiessen — APPROVED** [review]
Looks good and working as intended. :+1:

**Pull request merged**